indico/modules/files/controllers.py

```diff
@@ -29,7 +30,7 @@ class UploadFileMixin:
 
     @use_kwargs({'file': fields.Field(required=True)}, location='files')
     def _process(self, file):
-        if not self.validate_file(file):
+        if not (validate_upload_file_size(file) and self.validate_file(file)):
```

Comment:
This is much more readable:

```suggestion
        if not validate_upload_file_size(file) or not self.validate_file(file):
```